datasets/api.py

```diff
@@ -144,15 +144,32 @@ def perform_create(self, serializer):
         serializer.save(schema=schema)
 
 
+class DimensionSerializer(I18nFieldSerializerMixin, serializers.ModelSerializer):
+    categories = DimensionCategorySerializer(many=True, required=False)
+    name = serializers.CharField(source='name_i18n')
+
+    class Meta:
+        model = Dimension
+        fields = ['uuid', 'name', 'categories']
+
+
+class DatasetSchemaDimensionSerializer(serializers.ModelSerializer):
+    dimension = DimensionSerializer(many=False, required=False)
+
+    class Meta:
+        model = DatasetSchemaDimension
+        fields = ['order', 'dimension']
+
+
 class DatasetSchemaSerializer(I18nFieldSerializerMixin, serializers.ModelSerializer):
-    dimension_categories = DatasetSchemaDimensionCategorySerializer(
+    dimensions = DatasetSchemaDimensionSerializer(
         many=True, required=False,
     )
     metrics = DatasetMetricSerializer(many=True, required=False, read_only=True)
 
     class Meta:
         model = DatasetSchema
-        fields = ['uuid', 'time_resolution', 'unit', 'name', 'dimension_categories', 'metrics', 'start_date']
+        fields = ['uuid', 'time_resolution', 'unit', 'name', 'dimensions', 'metrics', 'start_date']
```

Comment:
These changes change the API response, something probably got broken (I'm not entirely sure what). How should we follow up to fix that?

Reply:
Yeah, we need to adapt the grid editor api adapter class a little bit, no biggie.

Reply:
I attempted to do that in this PR, hope I didn't miss anything: https://github.com/kausaltech/kausal-extensions/pull/73